admin/tests/acceptance/authenticated/users/get-test.js

```diff
@@ -136,12 +138,12 @@ module('Acceptance | authenticated/users/get', function (hooks) {
       });
       const expectedAuthenticationMethodCountBeforeAnonymisation = 3;
       const expectedAuthenticationMethodCountAfterAnonymisation = 0;
-
+      const connections = this.intl.t('pages.user-details.navbar.connections');
```

Comment:
Une suggestion pour un nommage qui serait un peu plus indicatif :
```suggestion
      const connectionTabLabel = this.intl.t('pages.user-details.navbar.connections');
```